packages/dara-core/dara/core/persistence.py

```diff
@@ -76,10 +142,47 @@ def dict(self, *args, **kwargs):
 class BackendStore(PersistenceStore):
     """
     Persistence store implementation that uses a backend implementation to store data server-side
+
+    :param uid: unique identifier for this store; defaults to a random UUID
+    :param backend: the backend to use for storing data; defaults to an in-memory backend
+    :param scope: the scope for the store; if 'global' a single value is stored for all users,
+        if 'user' a value is stored per user
     """
 
     uid: str = Field(default_factory=lambda: str(uuid4()))
     backend: PersistenceBackend = Field(default_factory=InMemoryBackend, exclude=True)
+    scope: Literal['global', 'user'] = 'global'
```

Comment:
For the type here, for example with cache we use CacheType being a enum but also accepting their string equivalent. Do we want to do the same here?

Reply:
This is not equivalent to `CacheType` as we don't have a `session` option available. If you're asking whether we should introduce an enum for those, I think it's overkill for this simple of a case and strings should be enough